pre-build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+apt-get update
+apt-get install -y debootstrap debian-archive-keyring jq dpkg-dev
```

Comment:
I just realised that minideb can only be built on a debian based distro. 
I suggest adding a check here to error out with a useful message when the host platform is not a debian based distro, maybe:

```bash
if [[ ! -f /etc/debian_version ]]; then
  echo "minideb can currently only be built on debian based distros, aborting..."
  exit 1
fi
```